app/assets/stylesheets/resources.css.scss

```diff
@@ -192,36 +195,31 @@
   }
 
   @media (max-width: 990px) {
-    margin-top: 100px;
+    margin-top: 90px;
 
     .resources-sidebar {
-      ul {
-        padding: 0;
-      }
-
-      li.selected a {
-        margin: 0;
-        width: inherit;
-      }
+      display: none;
+    }
 
-      li, li.selected {
-        width: inherit;
-        margin: 0 5px 10px 0;
-        display: inline-block;
-        padding: 8px;
-        border: 1px solid $light_grey;
-      }
+    .dropdown {
```

Comment:
If we remove `<li>` from the html, we'll need to update css to have styling consistent with the code base:
```
.dropdown-menu {
        margin-left: 5px;

        a {
          display: block;
          padding: 8px;
          font-family: ArcherBook;
          color: $black;

          &.selected {
            background-color: $yellow;
            color: white;
          }

          &:hover{
            background-color: $yellow;
            text-decoration: none;
          }
        }
      }
```

Reply:
done